Packs/MicrosoftGraphUser/Integrations/MicrosoftGraphUser/MicrosoftGraphUser.yml

```diff
@@ -410,25 +410,90 @@ script:
       type: String
     - contextPath: MSGraphUserDirectReports.Reports.UserPrincipalName
       description: The user principal name (UPN) of the user. The UPN is an Internet-style login name for the user based on the Internet standard RFC 822. By convention, this should map to the user's email name. The general format is alias@domain, where domain must be present in the tenant’s collection of verified domains. This property is required when a user is created. The verified domains for the tenant can be accessed from the verifiedDomains property of organization.
       type: String
+  - arguments:
+    - default: false
+      description: The User ID or userPrincipalName of the user for which to get the manager properties.
+      isArray: false
+      name: user
+      required: true
+      secret: false
+    deprecated: false
+    description: "Retrieves the properties from the manager of a user."
+    execution: false
+    name: msgraph-user-get-manager
+    outputs:
+    - contextPath: MSGraphUserManager.ID
+      description: User's user ID.
+      type: String
+    - contextPath: MSGraphUserManager.Manager.ID
+      description: Manager's user ID.
+      type: String
+    - contextPath: MSGraphUserManager.Manager.DisplayName
+      description: User's display name.
+      type: String
+    - contextPath: MSGraphUserManager.Manager.GivenName
+      description: User's given name.
+      type: String
+    - contextPath: MSGraphUserManager.Manager.BusinessPhones
+      description: User's business phone numbers.
+      type: String
+    - contextPath: MSGraphUserManager.Manager.JobTitle
+      description: User's job title.
+      type: String
+    - contextPath: MSGraphUserManager.Manager.Mail
+      description: User's mail address.
+      type: String
+    - contextPath: MSGraphUserManager.Manager.MobilePhone
+      description: User's mobile phone number.
+      type: String
+    - contextPath: MSGraphUserManager.Manager.OfficeLocation
+      description: User's office location.
+      type: String
+    - contextPath: MSGraphUserManager.Manager.PreferredLanguage
+      description: User's preferred language.
+      type: String
+    - contextPath: MSGraphUserManager.Manager.Surname
+      description: User's surname.
+      type: String
+    - contextPath: MSGraphUserManager.Manager.UserPrincipalName
+      description: User's principal name.
+      type: String
+  - arguments:
+    - default: false
+      description: User ID or userPrincipalName of the user to which to assign a manager.
+      isArray: false
+      name: user
+      required: true
+      secret: false
+    - default: false
+      description: User ID or userPrincipalName of the manager.
+      isArray: false
+      name: manager
+      required: true
+      secret: false
+    deprecated: false
+    description: "Assigns a manager to the specified user. Permissions: - User.ReadWrite (Delegated & Application)"
```

Comment:
@JT-NL are you sure both permissions, delegated and application are required? i think application is enough

Reply:
@Itay4  No, only one of them is [required](https://docs.microsoft.com/en-us/graph/api/user-post-manager?view=graph-rest-1.0&tabs=http). I updated the description to make that clear.